### DISCUSSION TITLE
Using Custom Icons

### DISCUSSION THREAD

**Discussion by chilleregeravi** [vuepress-theme-hope/vuepress-theme-hope#2724] [Q&A] [answered]
I have my own set of icons with a style.css. How can I go about using them? I have not created them on fontawesome.

**Comment by Mister-Hope** [comment, marked as answer]
Set iconPrefix to empty string manually and add icon class yourself

**Reply by chilleregeravi** [reply]
Would this be possible to add as a feature to the theme?

**Reply by Mister-Hope** [reply]
Why isn't it a feature of theme now? I already gives you iconAssets and iconPrefix option. Just add your css link to iconAssets with a string array, iconPrefix should be empty string.

```ts
hopeTheme({
  // the auto infered iconPrefix for this should be an empty string already
   iconAssets: [
     'fontawesome',
     '<base>/link/to/your.css',
   ]
});
```

**Reply by Mister-Hope** [reply]
If you do think you have any troubles, open a new pr with reproduction

**Reply by chilleregeravi** [reply]
I get the below error when configuring it 
![image](https://user-images.githubusercontent.com/640369/218411746-e37314f5-59de-4a1f-abfc-6445373bb25d.png)

It says Type 'font-awesome' is not assignable to a link

**Reply by chilleregeravi** [reply]
@Mister-Hope  - Could you help with the above issue?

**Reply by Mister-Hope** [reply]
Check my reply again, also you should insure that the links is available in server

**Reply by chilleregeravi** [reply]
yes, it's working now... :), thanks, there was a problem on my side in understanding.
